app.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const app = express();
+const endpointsJson = require('./endpoints.json')
+//const getApi = require('./controllers/api-controller')
+
+app.get('/api', (req, res) => {
+    res.status(200).send({ endpoints: endpointsJson})
+})
```

Comment:
This is a good start, but your comment suggests you thought about extracting this out into a controller, which I agree with, so how would you extract this out?